scripts/in_container/prod/entrypoint_prod.sh

```diff
@@ -111,6 +111,19 @@ else
     verify_db_connection "${AIRFLOW__CORE__SQL_ALCHEMY_CONN}"
 fi
 
+# The Bash and python commands still should verify the basic connections so they are run after the
```

Comment:
I'm not sure I agree with this -- if we aren't running airflow, we likely want to do something else, often related to debugging.


Hmm.

Reply:
Oh this didn't actually change behaviour/in fact it's better now. NM :)

Reply:
Yeah. you can always NOT pass the SQL variables.

Reply:
And this is written now in this sequence in the docs -> I updated the "what happens in entrypoint docs and made sure that the sequence of things described there corresponds to the entrypoint script